ubuntu/Dockerfile

```diff
@@ -1,6 +1,5 @@
 ARG UBUNTU_RELEASE
 FROM ubuntu:${UBUNTU_RELEASE}
-ARG UBUNTU_RELEASE
```

Comment:
hmmm does this work? See https://github.com/moby/moby/issues/34129

Reply:
Ah, thanks - reverted.